views/package.json

```diff
@@ -8,6 +8,6 @@
     "url": "git+https://github.com/oat-sa/extension-tao-items.git"
   },
   "dependencies": {
-    "@oat-sa/tao-item-runner": "0.4.0"
+    "@oat-sa/tao-item-runner": "0.6.0"
```

Comment:
Why not `0.6.1`, fixing a rendering issue https://github.com/oat-sa/tao-item-runner-fe/pull/26?